hack/run-functests.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+source hack/common.sh
+
+NO_COLOR=""
```

Comment:
Is this part really necessary?
To me, it looks like it's just some additional burden on the eyes when reading the code.

Reply:
it can be reintroduced later, but I see the value here. If the tests are trying to color the logs while running on an environment which doesn't support so (CI), the final result is almost unreadable.